### PR TITLE
#1959 - Staging sims prod environment formio

### PR DESCRIPTION
**Pull request by guru-aot**
Formio permission change in the dockerfile while build. 

The tried combinations are 640, 644, 650, 654, 750, 754, 770, 774 and 777. So with the least permission level of 770, 774 and 777 which worked. Selected 770.

### CODE REVIEW

File: sources/packages/forms/Dockerfile [hunk 1]
```diff
@@ -48,7 +48,7 @@ RUN ln -sf $NPM_PACKAGES/node_modules node_modules
 ENV DEBUG=""
 
 RUN set -x \
-  && chmod -R 777 /app/
+  && chmod -R 770 /app/
```
Comment (dheepak-aot) [inline sources/packages/forms/Dockerfile:51]: Asking for clarification?

What is the standard that we are trying to comply with here (by changing permission)?

Sonar? or any security vulnerability issue from scans previously?

Comment (sh16011993) [inline sources/packages/forms/Dockerfile:51]: Shouldn't the group have only read and execute permissions?

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the change. Looks good 👍 
For history, please just mention in the PR description that other permission like 750 impacted form.io server execution.

File: sources/packages/forms/Dockerfile (hunk 1 above)
Comment (guru-aot) [reply sources/packages/forms/Dockerfile:51]: yes i tried with 750, but it didnt start the application. The write is required for the group to create users in the server, i guess. The tried combinations are 640, 644, 650, 654, 750, 754, 770, 774 and 777. So with the least permission level of 770, 774 and 777 which worked. Selected 770.

Comment (andrewsignori-aot) [reply sources/packages/forms/Dockerfile:51]: We are trying to be less permissive as possible and still have the form.io server running.
image

Comment (guru-aot) [reply sources/packages/forms/Dockerfile:51]: Security vulnerability to restrict others to have any permission other than the owner and group.

**Review by sh16011993 — APPROVED** [review]
LGTM 👍

**Review by ann-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — APPROVED** [review]
👍

**Pull request merged**